.storybook/main.js

```diff
@@ -9,7 +9,6 @@ module.exports = {
     "@storybook/addon-essentials",
     "@storybook/addon-interactions",
     "@storybook/addon-a11y",
-    "storybook-i18n/preset",
```

Comment:
I noticed this addon not working nice with the docs addon. I removed it and replaced its features in the `preview.js` directly.